solutions/swb-app/src/users/createUserRequest.ts

```diff
@@ -0,0 +1,17 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+
+import { z } from 'zod';
+
+// eslint-disable-next-line @rushstack/typedef-var
+export const CreateUserRequestParser = z
+  .object({
+    firstName: z.string().min(1),
+    lastName: z.string().min(1),
+    email: z.string().min(1)
```

Comment:
Nit: did we create a backlog item to specify string length (i.e. max) for these parameters for the User APIs?

Reply:
yes we have items to adress this, I did it in this PR because I was parsing the required from the schema, but there is a backlog item to make sure we have the correct required, length and regex if needed